cpp/include/cuspatial/spatial_join.hpp

```diff
@@ -51,44 +52,97 @@ namespace cuspatial {
  * poly_offset - INT32 column of indices for each poly bbox that intersects with the quadtree.
  * quad_offset - INT32 column of indices for each leaf quadrant intersecting with a poly bbox.
  */
-
 std::unique_ptr<cudf::table> quad_bbox_join(
-  cudf::table_view const &quadtree,
-  cudf::table_view const &poly_bbox,
+  cudf::table_view const& quadtree,
+  cudf::table_view const& poly_bbox,
   double x_min,
   double x_max,
   double y_min,
   double y_max,
   double scale,
   int8_t max_depth,
-  rmm::mr::device_memory_resource *mr = rmm::mr::get_default_resource());
+  rmm::mr::device_memory_resource* mr = rmm::mr::get_default_resource());
 
 /**
- * @brief pair points and polygons using ray-cast based point-in-polygon test algorithm in two
- *phases: phase 1 counts the total number of output pairs for precise memory allocation phase 2
- *actually writes (point,polygon) pairs
- *
- * @param pq_pair: table of two arrays for (quadrant,polygon) pairs derived from spatial filtering
- * @param quadtree: table of five arrays derived from quadtree indexing on points (key,lev,sign,
- * length, fpos)
- * @param pnt: table of two arrays for points (x,y). note that points are in-place sorted in
- * quadtree construction and have different orders than the orginal input points.
- * @param fpos: feature/polygon offset array to rings
- * @param rpos: ring offset array to vertex
- * @param poly_x: polygon x coordiante array.
- * @param poly_y: polygon y coordiante array.
- *
- * @return array of (polygon_index, point_index) pairs that point is within polyon;
- * point_index and polygon_index are offsets of point and polygon arrays, respectively
- */
-// std::unique_ptr<cudf::table> pip_refine(
-//   cudf::table_view const& pq_pair,
-//   cudf::table_view const& quadtree,
-//   cudf::table_view const& pnt,
-//   cudf::column_view const& poly_fpos,
-//   cudf::column_view const& poly_rpos,
-//   cudf::column_view const& poly_x,
-//   cudf::column_view const& poly_y,
-//   rmm::mr::device_memory_resource* mr = rmm::mr::get_default_resource());
+ * @brief Finds points in a set of (polygon, quadrant) pairs derived from spatial filtering.
```

Comment:
This brief is unclear to me. What points does it find? What does "derived from spatial filtering" mean?

Reply:
Great point. "derived from spatial filtering" means "the table returned by `quad_bbox_join()`", but I'll modify this to be more clear.

Reply:
On a related note, I think the `quad_bbox_join` name could be more descriptive. Would that be good to add to this PR before it's released in 0.15?

Reply:
Probably yes.